upgrade/helpers/tasks.py

```diff
@@ -1150,3 +1150,19 @@ def mongo_db_engine_upgrade(upgrade_type):
     logger.info("MongoDB DataBase Engine Upgraded Successfully")
     logger.highlight('Time taken by MongoDB DataBase Engine Upgrade - {}'.format(
         str(postup_time - preup_time)))
+
+
+def foreman_packages_installation_check(state="unlock"):
+    """
+    This function is used to change the state of the foreman-package installation method,
+    And it will be applicable only if the PERFORM_FOREMAN_MAINTAIN_UPGRADE is False.
+
+    :param str state: To perform the installation using foreman-maintain the state will be
+    "lock" otherwise "unlock"
+    """
+    if os.environ.get('PERFORM_FOREMAN_MAINTAIN_UPGRADE') != 'true':
+        logger.info("{} the foreman-maintain packages".format(state))
+        run("foreman-maintain packages {}".format(state))
```

Comment:
add -y so that if utilitu not installed f-m should not wait

Reply:
fixed.